woocommerce/wc-admin
# Next new novel navigation nudge note

Fixes #6402 

Adds the latest nav note to nudge users to enable the new nav feature.

### Screenshots
<img width="594" alt="Screen Shot 2021-03-17 at 8 53 43 AM" src="https://user-images.githubusercontent.com/10561050/111470488-4c9b6680-86fe-11eb-9c14-a642b84d62a9.png">


### Detailed test instructions:

1. Download a plugin like WP Crontrol.
2. Turn on the new navigation under WooCommerce -> Settings -> Advanced.
3. Under Tools -> Cron Events, run the `wc_admin_daily` event.
4. Go to the homescreen, and note that the new note `You now have access to the WooCommerce navigation` is not added.
5. Turn off the new navigation.
6. Run `wc_admin_daily` again.
7. Go to the homescreen and see the new note.
8. Click `Enable` on the note actions.
9. Make sure the navigation has been enabled and you are redirected back to the screen where you viewed the note.
10. Delete the note if you have database access (or create a new site).
11. With navigation disabled, run the `wc_admin_daily` cron event.
12. Check that the note was added.
13. Manually turn on the note via WooCommerce -> Settings -> Advanced.
14. Check that the note was removed (actioned).
